docs/notebooks/expected_improvement.pct.py

```diff
@@ -28,15 +28,14 @@
 # %% [markdown]
 # ## Sample the observer over the search space
 #
-# Sometimes we don't have direct access to the objective function. We only have an observer that indirectly observes it. In _Trieste_, the observer outputs a number of datasets, each of which must be labelled so the optimization process knows which is which. In our case, we only have one dataset, the objective. We'll use _Trieste_'s default label for single-model setups, `OBJECTIVE`. We can convert a function with `branin`'s signature to a single-output observer using `mk_observer`.
+# Sometimes we don't have direct access to the objective function. We only have an observer that indirectly observes it. In _Trieste_, an observer can output a number of datasets. In our case, we only have one dataset, the objective. We can convert a function with `branin`'s signature to a single-output observer using `mk_observer`.
```

Comment:
we need to move this explanation somehow to the constraints notebook - and maybe expand it a bit, and make sure the "how to" questions point to the constraints notebook.

Reply:
I've just made a first stab at updating the constraints notebook, but will take a look at where else I can document it.